pagopa/pn-frontend
# PN-7453-componente-dialog-con-style-non-centralizzato

## Short description
Replaced some `Dialog` components with `PnDialog`.

## List of changes proposed in this pull request
- replaced `DialogActions` with `PnDialogActions`
- replaced `DialogContent` with `PnDialogContent`
- replaced `Dialog` with `PnDialog`
- moved `Dialog`'s props of `PnDialog` at the bottom in order to override default props
- adjusted relative tests after changes

## How to test
Just `yarn start` inside PA's package and check if dialogs are correctly rendered.
